### ISSUE TITLE
An accuracy problem when running Legion on only a single GPU

### ISSUE THREAD

**Issue by WWWzq-01** [RC4ML/Legion#3]
I set `--gpu-num` to 1 and then run the commands as described in README.md.
sampling:
<img width="484" alt="image" src="https://github.com/RC4ML/Legion/assets/52373880/b53aa930-00da-4480-a68a-914dea31d5e0">
training:
<img width="499" alt="image" src="https://github.com/RC4ML/Legion/assets/52373880/f668073f-fa3f-4003-86ff-8f1a2de73da7">

Why is the accuracy on both the validation set and test set 1.0?

**Comment by JIESUN233** [comment]
This is because many datasets, like ukunion, originally have no features and labels. I disable the feature/label loading for faster preprocessing.  I will open it in the other branch in a day.

**Comment by JIESUN233** [comment]
Hi, I enabled the feature/label loading in the branch: Evaluation-support. Please see the instruction changes in part 3 (dataset preparation) and part 5 (run legion).

**Comment by WWWzq-01** [comment]
That worked, thank you!
